tests/connection/test_connection_debug_logging.py

```diff
@@ -24,37 +24,45 @@ class TestConnectionDebugLogging(unittest.TestCase):
     ansi_escape = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
 
     # ============================================================================
-    # Add Connection
+    # Alter Connection
```

Comment:
```python
# Connect Connection
```

Reply:
fixed across the board as well as in group connection admin.